posix.mak

```diff
@@ -119,7 +119,13 @@ ifeq ($(VERBOSE_RDMD_TEST), 1)
 	override VERBOSE_RDMD_TEST_FLAGS:=-v
 endif
 
-test_rdmd: $(ROOT)/rdmd_test $(RDMD_TEST_EXECUTABLE)
+export RDMD_TEST_EXECUTABLE
+export ROOT
```

Comment:
Can we just export these variables when invoking the Makefile to avoid leakage and being explicit?